src/Microsoft.AspNet.Mvc.TagHelpers/Internal/GlobbingUtility.cs

```diff
@@ -0,0 +1,120 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNet.FileProviders;
+using Microsoft.AspNet.Http;
+using Microsoft.Framework.Cache.Memory;
+using Microsoft.Framework.FileSystemGlobbing;
+
+namespace Microsoft.AspNet.Mvc.TagHelpers.Internal
+{
+    /// <summary>
+    /// Utility methods for <see cref="ITagHelper"/>'s that support attributes containing file globbing patterns.
+    /// </summary>
+    public class GlobbingUtility
```

Comment:
We want to avoid generic names like Utility, Helper etc. in the codebase to the extent possible. (Yes I know we got HtmlHelper :)) Perhaps this should be `GlobbingUrlBuilder`?